packages/ext-gen/ext-gen.js

```diff
@@ -666,8 +666,8 @@ async function stepCreate() {
     console.log(boldRed('Error in npm install: ' + err));
   }
 
-  var frameworkPath = path.join(destDir, 'package.json');
-  var cmdPath = path.join(destDir, 'package.json');
+  var frameworkPath = path.join(destDir, 'node_modules', npmScope, 'ext', 'package.json');
```

Comment:
We are falling back to original code and for VS-code work once we identify common fix for we will update. until that will fallback to original/prv version of code.